### PR TITLE
Allows access to payload in callback validator

### PR DESCRIPTION
**Pull request by conradkleinespel**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #15092 |
| License | MIT |
| Doc PR | none |

### CODE REVIEW

File: src/Symfony/Component/Validator/Tests/Constraints/CallbackValidatorTest.php [hunk 1]
```diff
@@ -227,4 +226,28 @@ public function testAnnotationInvocationMultiValued()
 
         $this->assertEquals(new Callback(array(__CLASS__.'_Class', 'validateCallback')), $constraint);
     }
+
+    public function testPayload()
```
Comment (xabbuh) [inline src/Symfony/Component/Validator/Tests/Constraints/CallbackValidatorTest.php]: Imo this should be changed to something like `testPayloadIsPasswordToCallback` to reflect what we actually expect to happen in the test.

Comment (jakzal) [inline src/Symfony/Component/Validator/Tests/Constraints/CallbackValidatorTest.php]: @xabbuh you probably meant `testPayloadIsPassedToCallback()` ;)

Comment (xabbuh) [inline src/Symfony/Component/Validator/Tests/Constraints/CallbackValidatorTest.php]: How did you guess that? :D

Comment (conradkleinespel) [inline src/Symfony/Component/Validator/Tests/Constraints/CallbackValidatorTest.php]: Done. Thanks for the suggestion :+1:

**Comment by fabpot** [comment]
Thank you @conradkleinespel.

**Pull request merged**